TezosKit/Conseil/GetOriginatedAccountsRPC.swift

```diff
@@ -0,0 +1,27 @@
+// Copyright Keefer Taylor, 2019.
+
+/// An RPC which fetches originated accounts for an account.
+public class GetOriginatedAccounts: ConseilQueryRPC<[[String: Any]]> {
+  /// - Parameters:
+  ///   - account: The account to query.
+  ///   - limit: The number of items to return.
+  ///   - apiKey: The API key to send in the request headers.
+  ///   - platform: The platform to query.
+  ///   - network: The network to query.
+  public init?(account: String, limit: Int, apiKey: String, platform: ConseilPlatform, network: ConseilNetwork) {
+    let predicates: [ConseilPredicate] = [
+      ConseilQuery.Predicates.predicateWith(field: "manager", set: [account])
```

Comment:
One thing to note is that 3rd parties can originate accounts on your behalf making you manager.

Reply:
That should still be covered in this use case right?